duckduckgo/Android
# Show scrollbars on webview. Also, moved webview config back to XML.

Asana Issue URL: https://app.asana.com/0/414730916066338/547397654936878
GitHub Issue: #170 

## Description
Add scrollbars to the webview.


## Steps to Test this PR:
1. Visit arstechnica.com to ensure scrollbar visible against black themed webpage
1. Visit medium.com to ensure visible against white theme webpage
1. Ensure scroll bar only visible when scrolling, and fades away when not scrolling
